From 5e2f9a4ea355e6785fb021dd34cc8cd666e655d9 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@gmail.com>
Date: Wed, 4 Feb 2026 11:08:03 +0100
Subject: [PATCH] opencpn.appdata: Add vcs.browser url (flathub linter warning)

---
 resources/opencpn.appdata.xml.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/resources/opencpn.appdata.xml.in b/resources/opencpn.appdata.xml.in
index bd4f28e0f..ba9831d65 100644
--- a/resources/opencpn.appdata.xml.in
+++ b/resources/opencpn.appdata.xml.in
@@ -50,6 +50,7 @@
     </screenshot>
   </screenshots>
   <url type="homepage">https://opencpn.org/</url>
+  <url type="vcs-browser">https://github.com/opencpn/OpenCPN/</url>
   <update_contact>https://opencpn.org/flyspray/</update_contact>
   <content_rating type="oars-1.0" />
 </component>
-- 
2.52.0

