From 2e5834eb11161c9eddaedb0ef169fcb35a37f0d1 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@nowhere.net>
Date: Mon, 5 Feb 2024 16:39:27 +0100
Subject: [PATCH] opencpn.appdata: Add missing tag (#3626)

---
 resources/opencpn.appdata.xml.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/opencpn.appdata.xml.in b/data/opencpn.appdata.xml.in
index daabb9017..40da7955b 100644
--- a/data/opencpn.appdata.xml.in
+++ b/data/opencpn.appdata.xml.in
@@ -5,6 +5,7 @@
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
   <name>OpenCPN</name>
+  <developer_name>The OpenCPN community</developer_name>
   <summary>A concise Chartplotter and Navigation software</summary>
   <summary xml:lang="sv">Kartplotter och navigationsprogram</summary>
   <provides>
-- 
2.43.0

