diff --git a/cmake/TargetSetup.cmake b/cmake/TargetSetup.cmake
index b394af04c..e83466c9f 100644
--- a/cmake/TargetSetup.cmake
+++ b/cmake/TargetSetup.cmake
@@ -4,7 +4,15 @@
 
 if (OCPN_FLATPAK)
     set(PKG_TARGET "flatpak")
-    set(PKG_TARGET_VERSION "18.08")    # As of flatpak/*yaml
+    set(manifest_path "${PROJECT_SOURCE_DIR}/flatpak/org.opencpn.OpenCPN.yaml")
+    file(READ ${manifest_path} manifest)
+    string(REPLACE "\n" ";" manifest_lines "${manifest}")
+    foreach (_line ${manifest_lines})
+      if (${_line} MATCHES  "^runtime-version")
+        string(REGEX REPLACE ".*:" "" PKG_TARGET_VERSION "${_line}")
+      endif ()
+    endforeach ()
+    message(STATUS  "Building for flatpak runtime ${PKG_TARGET_VERSION}")
 elseif (MINGW)
     set(PKG_TARGET "mingw")
     if (CMAKE_SYSTEM_VERSION)
