diff --git a/data/opencpn.appdata.xml.in b/data/opencpn.appdata.xml.in
index ce1552905..5e175a8b5 100644
--- a/data/opencpn.appdata.xml.in
+++ b/data/opencpn.appdata.xml.in
@@ -42,7 +42,7 @@
   <screenshots>
     <screenshot type="default">
       <caption>Main window</caption>
-      <image>http://download.opencpn.org/screenshots/82434783-624dea00-9a61-11ea-847b-347d91b8f0fd.png</image>
+      <image>https://user-images.githubusercontent.com/405541/109418169-320b8280-79c7-11eb-8bf0-a7af3c533d8f.png</image>
     </screenshot>
   </screenshots>
   <url type="homepage">https://opencpn.org/</url>

