From e1f4eb6fddc14ff1054ab6f45217d43bc24fb4a8 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@nowhere.nt>
Date: Tue, 22 Jun 2021 11:09:14 +0200
Subject: [PATCH] Add warning message for aarch64

---
 config.h.in    | 2 ++
 src/chart1.cpp | 7 ++++++-
 2 files changed, 8 insertions(+), 1 deletion(-)

diff --git a/config.h.in b/config.h.in
index 4cd9ac70f..62dc643b0 100644
--- a/config.h.in
+++ b/config.h.in
@@ -37,6 +37,8 @@
 // The os compatible plugins are are built for
 #define PKG_TARGET "${PKG_TARGET}"
 
+#define PKG_TARGET_ARCH "${ARCH}"
+
 // The os version compatible plugins are are built for
 #define PKG_TARGET_VERSION "${PKG_TARGET_VERSION}"
 
diff --git a/src/chart1.cpp b/src/chart1.cpp
index 32d33f3fb..3d6229201 100644
--- a/src/chart1.cpp
+++ b/src/chart1.cpp
@@ -2607,7 +2607,12 @@ extern ocpnGLOptions g_GLOptions;
     gFrame->InitTimer.Start( 5, wxTIMER_CONTINUOUS );
     
     g_pauimgr->Update();
-    
+
+    std::string arch(PKG_TARGET_ARCH);
+    if (arch == "aarch64") {
+        OCPNMessageBox(0,
+           "This flatpak branch is not supported on aarch64, please use beta branch");
+    }
     return TRUE;
 }
 
-- 
2.30.2

