From a1efe823f146ed3324deebd1be0b06031f85e395 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@nowhere.nt>
Date: Sat, 13 Mar 2021 10:08:20 +0100
Subject: [PATCH 5/5] Update screenshot image URL.

---
 data/opencpn.appdata.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/opencpn.appdata.xml.in b/data/opencpn.appdata.xml.in
index ce1552905..5e175a8b5 100644
--- a/data/opencpn.appdata.xml.in
+++ b/data/opencpn.appdata.xml.in
@@ -42,7 +42,7 @@
   <screenshots>
     <screenshot type="default">
       <caption>Main window</caption>
-      <image>http://download.opencpn.org/screenshots/82434783-624dea00-9a61-11ea-847b-347d91b8f0fd.png</image>
+      <image>https://user-images.githubusercontent.com/405541/109418169-320b8280-79c7-11eb-8bf0-a7af3c533d8f.png</image>
     </screenshot>
   </screenshots>
   <url type="homepage">https://opencpn.org/</url>
-- 
2.29.2

