From a981742711f1bf724cc9f1d74598f5c766a3f392 Mon Sep 17 00:00:00 2001
From: Alec Leamas <leamas.alec@nowhere.net>
Date: Sun, 6 Feb 2022 15:39:12 +0100
Subject: [PATCH] Add -wx315 suffix to target ABI specification

---
 cmake/TargetSetup.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/TargetSetup.cmake b/cmake/TargetSetup.cmake
index 1aafc1145..427052bdb 100644
--- a/cmake/TargetSetup.cmake
+++ b/cmake/TargetSetup.cmake
@@ -5,7 +5,7 @@ if (NOT OCPN_TARGET_TUPLE STREQUAL "")
     list(GET OCPN_TARGET_TUPLE 0 PKG_TARGET)
     list(GET OCPN_TARGET_TUPLE 1 PKG_TARGET_VERSION)
 elseif (OCPN_FLATPAK)
-    set(PKG_TARGET "flatpak-${ARCH}")
+    set(PKG_TARGET "flatpak-${ARCH}-wx315")
     set(manifest_path "${PROJECT_SOURCE_DIR}/flatpak/org.opencpn.OpenCPN.yaml")
     file(READ ${manifest_path} manifest)
     string(REPLACE "\n" ";" manifest_lines "${manifest}")
-- 
2.34.1

