From 85d0e51ddbb1dee81756b6caaadb2f823a5645ec Mon Sep 17 00:00:00 2001
From: dsr <bdbcat@yahoo.com>
Date: Sat, 26 Feb 2022 20:17:02 -0500
Subject: [PATCH] Support scaled HDPI displays on GTK3 and MacOS/Retina

---
 src/glChartCanvas.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/glChartCanvas.cpp b/src/glChartCanvas.cpp
index f5916d53a..a9d505859 100644
--- a/src/glChartCanvas.cpp
+++ b/src/glChartCanvas.cpp
@@ -606,8 +606,8 @@ void glChartCanvas::Init() {
   m_gldc.SetGLCanvas(this);
 
   m_displayScale = 1.0;
-#ifdef __WXOSX__
-  // Support Mac Retina displays.
+#if defined(__WXOSX__) || defined(__WXGTK3__)
+  // Support scaled HDPI displays.
   m_displayScale = GetContentScaleFactor();
 #endif
 
-- 
2.34.1

